test/unit/support.js

```diff
@@ -56,18 +56,33 @@ testIframe(
 
 ( function() {
 	var expected,
+		version,
 		userAgent = window.navigator.userAgent,
 		expectedMap = {
-			edge: {},
-			ie_11: {},
-			chrome: {},
-			safari: {},
-			firefox: {},
-			ios: {}
+			edge: {
+				responseURL:true
```

Comment:
There should be a space after a colon. But this will be removed anyway as we don't need this support property.